utils/merkletrie/filesystem/node.go

```diff
@@ -0,0 +1,128 @@
+package filesystem
```

Comment:
doesn't make any sense at billy, is a very specific things to git, maybe outside of the merkletrie package.